changelog/11926.bugfix.md

```diff
@@ -0,0 +1 @@
+Decision to publish docs should not consider next major and minor alpha release versions.
```

Comment:
This changelog entry type should be of type `misc` since it's an internal maintenance fix for us.